db/seeds.rb

```diff
@@ -3,6 +3,9 @@
 AidRequest.destroy_all
 Organization.destroy_all
 
+ActiveRecord::Base.connection.reset_pk_sequence!('organizations')
+ActiveRecord::Base.connection.reset_pk_sequence!('aid_requests')
```

Comment:
these were running within the CSV `foreach` loop, so I took them out and put them at the top.

Reply:
Good catch!